study.md

```diff
@@ -151,7 +151,10 @@ What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
+GET - request data from server
+PUT - Add/update data on server
+DELETE - delete resource from server
+POST - request processing on server
```

Comment:
POST is used to create a resource while PUT and PATCH update or modify existing resources.